webapp/pages/post/_id/_slug/index.vue

```diff
@@ -125,12 +132,14 @@ export default {
       ready: false,
       title: 'loading',
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-03T17:26:44Z" title="Tuesday, December 3rd 2019, 6:26:44 pm +01:00">Dec 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -4,9 +4,35 @@
       :lang="post.language"
       v-if="post && ready"
       :image="post.image | proxyApiUrl"
-      :class="{ 'post-card': true, 'disabled-content': post.disabled }"
+      :class="{
+        'post-card': true,
+        'disabled-content': post.disabled,
+        'images-set-blur': post.checkedBlur,
+      }"
     >
-      <ds-space margin-bottom="small" />
+      <ds-text align="right" class="blurBox">
+        <ds-button
+          v-show="post.checkedBlur"
+          class="bluricon-post"
+          icon="eye"
+          primary
+          @click.prevent="unBlur"
+        ></ds-button>
+        <ds-button
+          v-show="blur"
```


```suggestion
          v-show="blurred"
```